client/src/pages/List.js

```diff
@@ -6,12 +6,15 @@ import { postProduct, fetchProductByname } from "../api/products";
 import useAsync from "../hooks/useAsync";
 import Header from "../components/Header";
 import PropTypes from "prop-types";
+import ProductAddContainer from "../components/ProductAddContainer";
+import FloatingActionButton from "../components/FloatingActionButton";
 
 function List({ onGroceryListChange }) {
   const { id } = useParams();
   const [query, setQuery] = useState([]);
   const [display, setDisplay] = useState(false);
   const [products, setProducts] = useState([]);
+  const [inputfield, setInputfield] = useState(false);
```

Comment:
```suggestion
  const [inputField, setInputField] = useState(false);
```

Just to follow your own naming convention it might be better to adjust the name.